cms/envs/common.py

```diff
@@ -766,6 +766,10 @@
     'notes',
 ]
 
+# XBlock types listed here will _always_ be selectable as Studio components
+XBLOCKS_ALWAYS_IN_STUDIO = [
```

Comment:
Yes. If you want XBLOCKS_ALWAYS_IN_STUDIO to be an instance specific list then I thought adding to aws.py was the way to go. Otherwise, your list may include xblocks that other instances have not installed?